ChangeLog

```diff
@@ -55,11 +55,16 @@
     * Fix: fixed crash when passing an empty string to cr_log_info (Ersikan).
     * Fix: fixed compilation errors on macOS (Kare Nuorteva, László Várady).
     * Fix: fixed --debug flag aborting runner with gdbserver >= 8.3.
+    * Fix: fixed gdb stepping issue with '--debug' on macOS (Apple silicon).
     * Misc: various documentation fixes (Oleksii Vilchanskyi, Sam Zaydel, Karim
       Dridi, Florent Poinsard, Tim Gates, László Várady).
     * Misc: various updates to dependencies (László Várady).
     * Misc: various build system fixes (Tomasz Sieprawski, László Várady).
     * Misc: various fixes to the autotools example (Bruno Belany).
+    * Misc: build system fixes and release automation.
```

Comment:
I don't think our users care much about the release automation to warrant an entry. The build system fixes should be covered by line 62.